core/modules/views/handlers/views_handler_field.inc

```diff
@@ -1232,7 +1232,7 @@ If you would like to have the characters \'[\' and \']\' please use the html ent
         $more_link_path = $this->options['alter']['more_link_path'];
         $more_link_path = strip_tags(decode_entities(strtr($more_link_path, $tokens)));
 
-        // Take sure that paths which was runned through url() does work as well.
+        // Make sure that the path which was run through url() works as well.
```

Comment:
```suggestion
        // Make sure that the path that was run through url() works as well.
```

This is a defining clause, so use "that." ("Which" clauses are always optional.)